README.md

```diff
@@ -1,12 +1,12 @@
 # React Native Animation Examples
 
-A collection of animations rendered with react-native using only JavaScript.
+A collection of animations rendered with React Native, using only JavaScript.
 
-This repo will mostly be examples taken from sites liek dribbble.com to brought to life using react-native.
+This repo will mostly be examples taken from sites like [Dribbble](https://dribbble.com/), and brought to life using React Native.
 
 ## Deployment
 
-This app uses [fastlane](https://docs.fastlane.tools/) for deployment. Fastlane allows for easy and reproducable deployments from the command line.
+This app uses [Fastlane](https://docs.fastlane.tools/) for deployment. Fastlane allows for easy and reproducable deployments from the command line.
```

Comment:
Hi @Saracevas! Thanks for the contribution.

Looking at the [fastlane website](https://fastlane.tools/) it looks like `fastlane` should remain lowercase.

Reply:
Ah my bad, @alexdunne! 

I appear to have been deceived by Google by accident:
![image](https://user-images.githubusercontent.com/9380319/73934209-f1c9f580-48d5-11ea-8bec-73410b64bb70.png)
 
It also seems like you have also spelled "fastlane" capitalised in the next sentence. Would you like me to fix that as well? I don't mind 👍

Reply:
I'm fairly sure Google's deceit wasn't on purpose!

That would be much appreciated.